visualization/FairnessDashboard/src/Controls/ModelComparisonChart.tsx

```diff
@@ -104,159 +123,241 @@ export class ModelComparisonChart extends React.PureComponent<IModelComparisonPr
+            props.data = ChartBuilder.buildPlotlySeries(props.data[0], data).map((series) => {
+                series.name = this.props.dashboardContext.modelNames[series.name];
+                series.text = this.props.dashboardContext.modelNames;
+                return series;
+            });
+
+            const accuracyMetricTitle = selectedMetric.title;
+            const parityMetricTitle = ParityOptions[this.props.parityPickerProps.selectedParityKey].title;
+            props.layout.xaxis.title = accuracyMetricTitle;
+            props.layout.yaxis.title = parityMetricTitle;
+
+            const InsightsIcon = () => <Icon iconName="CRMCustomerInsightsApp" className={styles.insightsIcon} />;
+            const DownloadIcon = () => <Icon iconName="Download" className={styles.downloadIcon} />;
+
+            const cancelIcon: IIconProps = { iconName: 'Cancel' };
+
+            mainChart = (
                 <div className={styles.main}>
-                    <div className={styles.chart}>
-                        <AccessibleChart
-                            plotlyProps={props}
-                            sharedSelectionContext={this.props.selections}
-                            theme={undefined}
-                        />
+                    <div className={styles.mainLeft}>
+                        <div className={styles.howTo}>
+                            <Modal
+                                titleAriaId="intro modal"
+                                isOpen={this.state.showModalIntro}
+                                onDismiss={this.handleCloseModalIntro}
+                                isModeless={true}
+                                containerClassName={styles.modalContentIntro}
+                            >
+                                <div style={{ display: 'flex' }}>
+                                    <IconButton
+                                        styles={iconButtonStyles}
+                                        iconProps={cancelIcon}
+                                        ariaLabel="Close intro modal"
+                                        onClick={this.handleCloseModalIntro}
+                                    />
+                                </div>
+                                <p className={styles.modalContentIntroText}>
+                                    {localization.ModelComparison.introModalText}
+                                </p>
+                                <div style={{ display: 'flex', paddingBottom: '20px' }}>
+                                    <PrimaryButton className={styles.doneButton} onClick={this.handleCloseModalIntro}>
+                                        {localization.done}
+                                    </PrimaryButton>
+                                </div>
+                            </Modal>
+                            <ActionButton onClick={this.handleOpenModalHelp}>
+                                <div className={styles.infoButton}>i</div>
```

Comment:
You could do this with the info icon as a button, to avoid adding the circle to the i text (which may not give the screen-reader behavior we'd like). See https://github.com/interpretml/interpret-community/blob/a6b72e535c67fb36746c77eb681cbb3a718d7ba3/visualization/dashboard/src/MLIDashboard/Controls/GlobalExplanationTab/GlobalExplanationTab.tsx#L351